arc/http/session/jwe.py

```diff
@@ -0,0 +1,82 @@
+import math
+import os
+import time
+from http import cookies
+from typing import Any, Dict
+
+from jwcrypto import jwe, jwk
+from jwcrypto.common import base64url_encode, json_decode, json_encode
+
+COOKIE_NAME: str = "_idx"
+
+
+def _get_key() -> str:
+    # 32 bit key size
```

Comment:
lol at my comment in the other repo, I think you established this indeed a 128bit / 16byte key size 🙃

Patch for that coming!